tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -29,12 +34,12 @@ def setUp(self):
         self.project2 = self.create_project()
         self.login_as(user=self.user)
 
-        self.transaction_lcp_metric = indexer.record(
+        self.transaction_lcp_metric = _indexer_record(
```

Comment:
This probably needs a `UseCaseKey.PERFORMANCE`

Reply:
The problem with that is that the read side also needs to be adjusted. So that means either changing class logic or test logic and I don't think I'm the right person to figure out what the right UseCaseKey is everywhere.